### DISCUSSION TITLE
Swap left and right layer keys = remote control ?

### DISCUSSION THREAD

**Discussion by auipga** [manna-harbour/miryoku#20] [Q&A] [answered]
## Imagine this scenario:

- You use bluetooth connection
- You run Netflix or you browser on your PC, but you're _far_ away from the desk.

## The solution:

Just take your right half.
Right thumb keys become TT("NAV"), TT("MOUSE") and TT("MEDIA")

With the NAV layer also containing <kbd>Ctrl+Tab</kbd> and <kbd>Ctrl+Shift+Tab</kbd> for switching tabs, <kbd>Ctrl</kbd>+(<kbd>Shift</kbd>)+<kbd>T</kbd>/<kbd>W</kbd>/... for (re)opening and closing tabs and maybe some other tweaks this would be perfect I think.
(This idea misses a solution for layer deactivation yet.)

**Comment by manna-harbour** [comment, marked as answer]
Thanks for the request!  I'm planning on adding custom layers (#17) and layer lock (#15), which together should cover your use case.
